src/main/java/springee/pet/PetController.java

```diff
@@ -20,7 +23,9 @@
     put(1, new Pet("Jerry", "Mouse", 1));
   }};
 
-  Integer counter = 2;
+  private Map<Integer, Pet> synPets = Collections.synchronizedMap(pets);
```

Comment:
this collection is thread-safe, but it is blocking. better choice: `ConcurrentHashMap`